### ISSUE TITLE
WPFUI.Theme.Manager.SetSystemTheme() throws System.NullReferenceException

### ISSUE THREAD

**Issue by MikeKatsoulakis** [lepoco/wpfui#9]
**Describe the bug**  
If the project has any ResourceDictionaries in the App.xaml that do not have the 'Source' property set, the above exception is being thrown.

**To Reproduce**  
1. Add a Resource dictionary to a demo/test project without the 'Source' property set.
2. Run it.

**Expected behavior**  
The Switch Method should ignore the Resource Dictionaries that do not have the 'Source' property set.

**Additional context**  
Tinkered a little with the code and adding a null check on the Switch Method of the Manager fixed this.
Basically `applicationDictionaries[i].Source != null`

**Comment by Pomianowski** [comment]
Hey @MikeKatsoulakis. Can you check if the bug still occurs in the latest prerelease?

**Comment by Lightczx** [comment]
the `CheckDictionarySource` in `GetAppTheme` can also throw `NullReferenceException`
both latest and prerelease would throw

My suggestion is provide a Property to manually set app theme and do the check if no value is set
